### DISCUSSION TITLE
[@types/p5] Type definition of module "p5/lib/addons/p5.sound" seems incorrect and doesn't work

### DISCUSSION THREAD

**Discussion by Nyaacinth** [DefinitelyTyped/DefinitelyTyped#63934] [Issues with a @types package] [answered]
The file with issue:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/p5/lib/addons/p5.sound.d.ts

This auto-generated file seems incorrect, we cannot declare modules with a relative path, and it doesn't work at all.

But if I change the declaration to:
```ts
declare namespace p5sound {
  // ...
}

export = p5sound
```
It will then work properly.

[a comment by an automated account is not shown]

**Comment by Nyaacinth** [comment]
Seems like there’s still no answer for this. Here’s the thing: I want to know how was that file generated, so that I can fix it. Or can I just ignore the comments and modify it?

**Comment by Zalastax** [comment, marked as answer]
Hi,

The file was generated using https://github.com/p5-types/p5.ts
@wuyudi has recently started maintenance of it again. I recommend creating an issue over there.

**Reply by Nyaacinth** [reply]
> Hi,
> 
> The file was generated using https://github.com/p5-types/p5.ts @wuyudi has recently started maintenance of it again. I recommend creating an issue over there.

Thanks for your reply! I’ll create an issue there.
